core/src/main/scala/kanaloa/reactive/dispatcher/queue/Worker.scala

```diff
@@ -235,16 +247,23 @@ object Worker {
   case object Working extends WorkerStatus
   case object WaitingToTerminate extends WorkerStatus
 
-  case class Hold(period: FiniteDuration)
+  case class SetDelay(period: Option[FiniteDuration])
   private[queue] case object WorkFinished
 
   class DefaultWorker(
```

Comment:
If you do decide to combine the QueueProcessor trait and DefaultQueuePRocossor class, I would argue we should do the same here, just to be consistent.